statistics/builder.go

```diff
@@ -156,10 +156,10 @@ func BuildColumn(ctx sessionctx.Context, numBuckets, id int64, collector *Sample
 
 // AnalyzeResult is used to represent analyze result.
 type AnalyzeResult struct {
-	TableID int64
-	Hist    []*Histogram
-	Cms     []*CMSketch
-	Count   int64
-	IsIndex int
-	Err     error
+	PhysicalID int64
```

Comment:
What is the PhysicalID?

Reply:
It is the partition id for a partitioned table, otherwise, it is the table id.

Reply:
Please add a comment for it.